spec/garage_client/request/pass_over_request_id_spec.rb

```diff
@@ -0,0 +1,40 @@
+require "spec_helper"
+
+describe GarageClient::Request::PassOverRequestId do
+  let(:client) do
+    GarageClient::Client.new
+  end
+
+  before do
+    Thread.current[:request_id] = 'request_id'
```

Comment:
you mean `Thread.current[:request_id]` should be restored to be an original value?

```ruby
around do |example|
  original = Thread.current[:request_id]
  Thread.current[:request_id] = 'request_id'
  example.run
  Thread.current[:request_id] =  original
end
```